<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="" />
        <meta name="author" content="" />
        <title>Ozkan Lab</title>
        <!-- <base href="https://cemkazan.github.io" /> -->
        <link rel="icon" type="image/x-icon" href="assets/favicon.ico" />
        <link href="../css/styles.css" rel="stylesheet" />
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
        <script src="../js/layout.js"></script>
        <style>
            .year-header {
                font-size: 1.5em;
                font-weight: bold;
            }
            .publication {
                margin-left: 20px;
            }
            .publications {
                display: block;
                margin-left: auto;
                margin-right: auto;
                width: 50%; /* Adjust this value to change the width of the content */
            }
        </style>
    </head>
    <body id="page-top" class="d-flex flex-column">
        <main class="mb-auto">
        <div id="loaded-layout"></div>
        <!-- Page-specific content goes here -->
        <section id="research" style="background-color: white;">
            <div class="container px-4">
                <!-- ... Existing content ... -->
            </div>
            <div class="publications">
                <div class="year-header">2024</div>
                <div class="publication">
                    <p><a href="https://journals.aps.org/prxlife/abstract/10.1103/PRXLife.2.013012" target="_blank">"RNA-induced Allosteric Coupling Drives Viral Capsid Assembly,"</a> S Hamilton, T Modi, P Šulc, SB Ozkan, PRX Life 2 (1), 013012</p>
                    <p><a href="https://www.cell.com/biophysj/abstract/S0006-3495(23)02009-X" target="_blank">"Enhancing deep learning model design through protein dynamics information integration,"</a> J Lu, ICJ Kazan, BK Kartchner, N Huynh, JH Mills, SB Ozkan, Biophysical Journal 123 (3), 207a-208a</p>
                    <p><a href="https://www.cell.com/biophysj/abstract/S0006-3495(23)01159-1" target="_blank">"Engineering gain-of-function mutants of a WW domain by dynamics and structural analysis,"</a> ICJ Kazan, J Lu, MI Rahman, N Halloran, AA Bobkov, SB Ozkan, Biophysical Journal 123 (3), 64a</p>
                    <p><a href="https://www.cell.com/biophysj/abstract/S0006-3495(23)01162-1" target="_blank">"Probing impact of allosteric (or distal) mutations on protein function through time-dependent linear response theory,"</a> P Campitelli, T Modi, SB Ozkan, Biophysical Journal 123 (3), 65a</p>
                </div>
                <div class="year-header">2023</div>
                <div class="publication">
                    <p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10515954/" target="_blank">"Some mechanistic underpinnings of molecular adaptations of SARS-COV-2 spike protein by integrating candidate adaptive polymorphisms with protein dynamics,"</a> NJ Ose, P Campitelli, T Modi, IC Kazan, S Kumar, SB Ozkan, bioRxiv</p>
                    <p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/pro.4759" target="_blank">"Engineering gain‐of‐function mutants of a WW domain by dynamics and structural analysis,"</a> J Lu, MI Rahman, IC Kazan, NR Halloran, AA Bobkov, SB Ozkan, Protein Science 32 (9), e4759</p>
                    <p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/pro.4700" target="_blank">"Allosteric regulatory control in dihydrofolate reductase is revealed by dynamic asymmetry,"</a> IC Kazan, JH Mills, SB Ozkan, Protein Science 32 (8), e4700</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(23)00069-3.pdf" target="_blank">"Protein dynamics provide mechanistic insights about epistasis among common missense polymorphisms,"</a> NJ Ose, P Campitelli, R Patel, S Kumar, SB Ozkan, Biophysical journal 122 (14), 2938-2947</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(22)02114-2.pdf" target="_blank">"Understanding the evolution of cold sensing and menthol activation in TRPM8,"</a> N Ramesh, D Luu, KH Shah, ICJ Kazan, SB Ozkan, WD Van Horn, Biophysical journal 122 (3), 197a</p>
                    <p><a href="https://journals.aps.org/prxlife/abstract/10.1103/PRXLife.2.013012" target="_blank">"RNA-induced allosteric coupling drives viral capsid assembly in the single-stranded RNA virus bacteriophage MS2,"</a> S Hamilton, T Modi, P Sulc, SB Ozkan, Biophysical Journal 122 (3), 48a-49a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(22)02079-3.pdf" target="_blank">"Dynamics-based network analysis identifies laci rheostats with high accuracy,"</a> P Campitelli, SB Ozkan, L Swint-Kruse, D Ross, Biophysical journal 122 (3), 189a-190a</p>
                    <p>Allosteric regulatory control in dihydrofolate reductase (DHFR) revealed with dynamic asymmetry in unliganded ensemble, ICJ Kazan, JH Mills, SB Ozkan, Biophysical Journal 122 (3), 48a</p>
                    <p><a href="https://pubs.acs.org/doi/abs/10.1021/acs.jpcb.2c05983" target="_blank">"Correlated Evolution of Low-Frequency Vibrations and Function in Enzymes,"</a> T Modi, P Campitelli, M Heyden, SB Ozkan, The Journal of Physical Chemistry B 127 (3), 616-622</p>
                    <p><a href="https://www.biorxiv.org/content/10.1101/2023.06.05.543665v1.abstract" target="_blank">"RNA-induced allosteric coupling drives viral capsid assembly in bacteriophage MS2,"</a> S Hamilton, T Modi, P Sulc, SB Ozkan, bioRxiv, 2023.06. 05.543665</p>
                    <p>Track: Protein and Ligand-A New Marriage Between an Old Couple Functional Ancestral Reconstruction of the Cold and Menthol Sensor TRPM8, D Luu, K Shah, N Ramesh, I Kazan, SB Ozkan, W Van Horn, PROTEIN SCIENCE 32</p>
                </div>
                <div class="year-header">2022</div>
                <div class="publication">
                    <p><a href="https://elifesciences.org/articles/67474" target="_blank">"Design of novel cyanovirin-N variants by modulation of binding dynamics through distal mutations,"</a> IC Kazan, P Sharma, MI Rahman, A Bobkov, R Fromme, G Ghirlanda, Elife 11, e67474</p>
                    <p><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.4474" target="_blank">"Investigating the allosteric response of the PICK1 PDZ domain to different ligands with all‐atom simulations,"</a> AO Stevens, IC Kazan, B Ozkan, Y He, Protein Science 31 (12), e4474</p>
                    <!-- <p><a href="URL_OF_THE_PUBLICATION" target="_blank">"Dynamic allostery highlights the evolutionary differences between the CoV-1 and CoV-2 main proteases,"</a> P Campitelli, J Lu, SB Ozkan, Biophysical journal 121 (8), 1483-1492</p> -->
                    <p> Dynamic allostery highlights the evolutionary differences between the CoV-1 and CoV-2 main proteases, P Campitelli, J Lu, SB Ozkan, Biophysical journal 121 (8), 1483-1492</p>
                    <p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010006" target="_blank">"Dynamic coupling of residues within proteins as a mechanistic foundation of many enigmatic pathogenic missense variants,"</a> NJ Ose, BM Butler, A Kumar, IC Kazan, M Sanderford, S Kumar, PLoS computational biology 18 (4), e1010006</p>
                    <p>"Understanding ligand specificity through protein network dynamics," P Campitelli, SB Ozkan, L Swint-Kruse, Biophysical Journal 121 (3), 51a </p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(21)01463-6.pdf" target="_blank">"Protein dynamics provide mechanistic insights about the epistatic relationships among highly observed potentially adaptive missense variants,"</a> NJ Ose, P Campitelli, RP Patel, SB Ozkan, S Kumar, Biophysical Journal 121 (3), 456a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(21)01462-4.pdf" target="_blank">"Vibrational density of states capture the role of dynamic allostery in protein evolution,"</a> T Modi, M Heyden, SB Ozkan, Biophysical Journal 121 (3), 456a</p>
                </div>
                <div class="year-header">2021</div>
                <div class="publication">
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(21)00324-6.pdf" target="_blank">"Can sequence-specific and dynamics-based metrics allow us to decipher the function in IDP sequences?"</a> SB Ozkan, Biophysical Journal 120 (10), 1857-1859</p>
                    <p><a href="https://www.nature.com/articles/s41467-021-22089-0" target="_blank">"Hinge-shift mechanism as a protein design principle for the evolution of β-lactamases from substrate promiscuity to specificity,"</a> T Modi, VA Risso, S Martinez-Rodriguez, JA Gavira, MD Mebrat, SB Ozkan, Nature Communications 12 (1), 1852</p>
                    <p><a href="https://www.mdpi.com/1422-0067/22/6/2895" target="_blank">"The role of rigid residues in modulating TEM-1 β-lactamase function and thermostability,"</a> B Kolbaba-Kartchner, IC Kazan, JH Mills, SB Ozkan, International Journal of Molecular Sciences 22 (6), 2895</p>
                    <p><a href="https://pubs.acs.org/doi/abs/10.1021/acs.jpcb.1c00364" target="_blank">"Local interactions that contribute minimal frustration determine foldability,"</a> T Zou, BW Woodrum, N Halloran, P Campitelli, AA Bobkov, G Ghirlanda, SB Ozkan, The Journal of Physical Chemistry B 125 (10), 2617-2626</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(20)32824-1.pdf" target="_blank">"Information Propagation in Time through Allosteric Signaling,"</a> T Modi, SB Ozkan, S Pressé, Biophysical Journal 120 (3), 300a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(20)32825-3.pdf" target="_blank">"Adaptive Evolution of SARS-CoV-2: A Perspective from Dynamics,"</a> IC Kazan, SB Ozkan, Biophysical Journal 120 (3), 300a-301a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(20)32822-8.pdf" target="_blank">"Allostery and Epistasis as Inherent Properties of Anisotropic Networks,"</a> P Campitelli, SB Ozkan, Biophysical Journal 120 (3), 300a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(20)32672-2.pdf" target="_blank">"Emerging Features of Rheostat Positions,"</a> L Swint-Kruse, AW Fenton, SB Ozkan, B Hagenbuch, P Campitelli, Biophysical Journal 120 (3), 277a-278a</p>
                    <p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0959440X20302116" target="_blank">"Protein folding stability and binding interactions through the lens of evolution: a dynamical perspective,"</a> T Modi, P Campitelli, IC Kazan, SB Ozkan, Current Opinion in Structural Biology 66, 207-215</p>
                    <p>"Dynamic Allostery Highlights the Evolutionary Differences between the CoV-1 and CoV-2 Main Proteases (preprint),"</a> P Campitelli, J Lu, SB Ozkan, 2021</p>
                    <p><a href="https://academic.oup.com/mbe/article/38/1/201/5891178" target="_blank">"Substitutions at nonconserved rheostat positions modulate function by rewiring long-range, dynamic interactions,"</a> P Campitelli, L Swint-Kruse, SB Ozkan, Molecular Biology and Evolution 38 (1), 201-214</p>
                </div>
                <div class="year-header">2020</div>
                <div class="publication">
                    <p><a href="https://www.biorxiv.org/content/10.1101/2020.12.21.423842v3.abstract" target="_blank">"Design of novel CV-N variants: Modulation of binding dynamics through distal mutations,"</a> IC Kazan, P Sharma, A Bobkov, R Fromme, G Ghirlanda, SB Ozkan, bioRxiv, 2020.12. 21.423842</p>
                    <p><a href="https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.2.023367" target="_blank">"Information propagation in time through allosteric signaling,"</a> T Modi, SB Ozkan, S Pressé, Physical Review Research 2 (2), 023367</p>
                    <p><a href="https://www.mdpi.com/1099-4300/22/6/667" target="_blank">"Allostery and epistasis: emergent properties of anisotropic networks,"</a> P Campitelli, SB Ozkan, Entropy 22 (6), 667</p>
                    <p><a href="https://www.annualreviews.org/content/journals/10.1146/annurev-biophys-052118-115517" target="_blank">"The role of conformational dynamics and allostery in modulating protein evolution,"</a> P Campitelli, T Modi, S Kumar, SB Ozkan, Annual Review of Biophysics 49, 267-288</p>
                    <p><a href="https://patents.google.com/patent/US20200069778A1/en" target="_blank">"Plant-expressed cocaine hydrolase variants of butyrylcholinesterase and methods of reducing cocaine-primed reinstatement,"</a> T Mor, K Larrimore, IJ Kazan, S Ozkan, J Neisewander, US Patent App. 16/560,993</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(19)31401-8.pdf" target="_blank">"Dynamic allosteric residue coupling reveals disease mechanism for Gaucher disease and nSNVs across the proteome,"</a> N Ose, BM Butler, A Kumar, SB Ozkan, S Kumar, Biophysical Journal 118 (3), 53a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(19)32163-0.pdf" target="_blank">"Large Scale Disease Susceptibility Prediction of nSNVs by Dynamics Information Based Neural Networks,"</a> IC Kazan, SB Ozkan, Biophysical Journal 118 (3), 205a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(19)31393-1.pdf" target="_blank">"Asymmetry in dynamic allosteric residue coupling (DARC) interactions captures evolutionary landscape,"</a> P Campitelli, SB Ozkan, L Swint-Kruse, Biophysical Journal 118 (3), 52a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(19)32162-9.pdf" target="_blank">"What are the Design Principles for Evolution from Promiscuous to Substrate Specificity?"</a> T Modi, VA Risso, S Martinez-Rodriguez, JA Gavira, SB Ozkan, Biophysical Journal 118 (3), 205a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(19)32736-5.pdf" target="_blank">"A method for the incorporation of protein dynamics into computational enzyme design using the Rosetta software suite,"</a> BK Kartchner, IC Kazan, SB Ozkan, JH Mills, Biophysical Journal 118 (3), 320a-321a</p>
                </div>
                <div class="year-header">2019</div>
                <div class="publication">
                    <p><a>"DESIGN OF NOVEL LECTINS BY COMPUTER-AIDED DIRECTED EVOLUTION,"</a> P Sharma, IC Kazan, SB Ozkan, G Ghirlanda, Protein Science 28, 81-81</p>
                    <p><a>"Conformational dynamics play critical role in protein evolution,"</a> SB Ozkan, Abstracts of Papers of the American Chemical Society 258</p>
                    <p><a href="https://link.springer.com/chapter/10.1007/978-3-030-30363-1_9" target="_blank">"Allostery and structural dynamics in protein evolution,"</a> P Campitelli, SB Ozkan, Evolution, Origin of Life, Concepts and Methods, 179-194</p>
                </div>
                <div class="year-header">2018</div>
                <div class="publication">
                    <p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006626" target="_blank">"Coevolving residues inform protein dynamics profiles and disease susceptibility of nSNVs,"</a> BM Butler, IC Kazan, A Kumar, SB Ozkan, PLoS Computational Biology 14 (11), e1006626</p>
                    <p><a href="https://www.mdpi.com/1422-0067/19/12/3808" target="_blank">"Mutations utilize dynamic allostery to confer resistance in TEM-1 β-lactamase,"</a> T Modi, SB Ozkan, International Journal of Molecular Sciences 19 (12), 3808</p>
                    <p><a>"Plant-expressed cocaine hydrolase variants of butyrylcholinesterase exhibit altered allosteric effects of cholinesterase activity and increased inhibitor sensitivity,"</a> KE Larrimore, I Kazan, L Kannan, R Kendle, T Jamal, M Barcus, A Bolia, SB Ozkan, Scientific Reports 8</p>
                    <p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0959440X18300071" target="_blank">"Biotechnological and protein-engineering implications of ancestral protein resurrection,"</a> VA Risso, JM Sanchez-Ruiz, SB Ozkan, Current Opinion in Structural Biology 51, 106-115</p>
                    <p><a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=Y4mtEasAAAAJ&cstart=20&pagesize=80&sortby=pubdate&citation_for_view=Y4mtEasAAAAJ:2P1L_qKh6hAC" target="_blank">"Design of Novel Lectins by Computer‐Guided Directed Evolution,"</a> P Sharma, IC Kazan, SB Ozkan, G Ghirlanda, The FASEB Journal 32, 673.24-673.24</p>
                    <p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0304394018301435" target="_blank">"A novel modelling mechanism of PAEL receptor and GABARAPL2 interaction involved in Parkinson's disease,"</a> P Dutta, L Dargahi, KE O'Connell, A Bolia, B Ozkan, AW Saile, KK Dev, Neuroscience Letter 673, 12-18</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(17)33449-5.pdf" target="_blank">"Allostery and Conformational Dynamics in Protein Evolution,"</a> SB Ozkan, Biophysical Journal 114 (3), 400a-401a</p>
                    <p><a href="https://pubs.acs.org/doi/abs/10.1021/acs.jpcb.7b11971" target="_blank">"Hinge-shift mechanism modulates allosteric regulations in human Pin1,"</a> P Campitelli, J Guo, HX Zhou, SB Ozkan, The Journal of Physical Chemistry B 122 (21), 5623-5629</p>
                    <p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6237863/" target="_blank">"Author Correction: Plant-expressed cocaine hydrolase variants of butyrylcholinesterase exhibit altered allosteric effects of cholinesterase activity and increased inhibitor sensitivity,"</a> KE Larrimore, IC Kazan, L Kannan, RP Kendle, T Jamal, M Barcus, SB Ozkan, Scientific Reports 8</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(17)32519-5.pdf" target="_blank">"Ancient thioredoxins evolved to modern-day stability–function requirement by altering native state ensemble,"</a> T Modi, J Huihui, K Ghosh, SB Ozkan, Philosophical Transactions of the Royal Society B</p>
                </div>
                <div class="year-header">2017</div>
                <div class="publication">
                    <p><a href="https://www.nature.com/articles/s41598-017-10571-z" target="_blank">"Plant-expressed cocaine hydrolase variants of butyrylcholinesterase exhibit altered allosteric effects of cholinesterase activity and increased inhibitor sensitivity,"</a> KE Larrimore, IC Kazan, L Kannan, RP Kendle, T Jamal, M Barcus, Scientific Reports 7 (1), 10419</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(16)32750-3.pdf" target="_blank">"Dynamic Flexibility Index Sheds Light on Pin1 Allostery,"</a> P Campitelli, HX Zhou, G Ghirlanda, SB Ozkan, Biophysical Journal 112 (3), 317a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(16)32182-8.pdf" target="_blank">"Intrinsically Disordered Protein Dynamics Uncovered through Dynamic Flexibility Index (DFI) Analysis,"</a> T Modi, GH Zerze, J Mittal, SM Vaiana, SB Ozkan, Biophysical Journal 112 (3), 208a-209a</p>
                </div>
                <div class="year-header">2016</div>
                <div class="publication">
                    <p><a href="https://pubs.acs.org/doi/abs/10.1021/acs.jcim.5b00587" target="_blank">"Adaptive BP-dock: an induced fit docking approach for full receptor flexibility,"</a> A Bolia, SB Ozkan, Journal of Chemical Information and Modeling 56 (4), 734-746</p>
                </div>
                <div class="year-header">2015</div>
                <div class="publication">
                    <p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0959440X1500158X" target="_blank">"Integration of structural dynamics and molecular evolution via protein interaction networks: a new era in genomic medicine,"</a> A Kumar, BM Butler, S Kumar, SB Ozkan, Current Opinion in Structural Biology 35, 135-142</p>
                    <p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.24947" target="_blank">"Partial unfolding and refolding for structure refinement: A unified approach of geometric simulations and molecular dynamics,"</a> A Kumar, P Campitelli, MF Thorpe, SB Ozkan, Proteins: Structure, Function, and Bioinformatics 83 (12), 2279-2292</p>
                    <p><a href="https://pubs.acs.org/doi/abs/10.1021/acs.biochem.5b00635" target="_blank">"A rigid hinge region is necessary for high-affinity binding of dimannose to cyanovirin and associated constructs,"</a> Z Li, A Bolia, JD Maxwell, AA Bobkov, G Ghirlanda, SB Ozkan, Biochemistry 54 (46), 6951-6960</p>
                    <p><a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=Y4mtEasAAAAJ&cstart=20&pagesize=80&sortby=pubdate&citation_for_view=Y4mtEasAAAAJ:M3NEmzRMIkIC" target="_blank">"Evolutionary hinge migration sheds light on the mechanism of green-to-red photoconversion in GFP-like proteins,"</a> RM Wachter, SB Ozkan, H Kim, T Zhou, Protein Science 24, 168-168</p>
                    <p>"Dynamics and Allostery in Ferritin," A Kumar, T Glembo, S Ozkan, Dynamics</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(15)00672-4.pdf" target="_blank">"The role of conformational dynamics and allostery in the disease development of human ferritin,"</a> A Kumar, TJ Glembo, SB Ozkan, Biophysical Journal 109 (6), 1273-1281</p>
                    <p><a href="https://www.tandfonline.com/doi/full/10.1080/07391102.2015.1032811" target="_blank">"Mechanism of protein evolution: conformational dynamics and allostery,"</a> T Zou, A Kumar, BM Butler, SB Ozkan, Journal of Biomolecular Structure and Dynamics 33 (sup1), 114-114</p>
                    <p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.24748" target="_blank">"Conformational dynamics of nonsynonymous variants at protein interfaces reveals disease association,"</a> BM Butler, ZN Gerek, S Kumar, SB Ozkan, Proteins: Structure, Function, and Bioinformatics 83 (3), 428-435</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(14)04058-2.pdf" target="_blank">"Physics Based Structure Refinement in Casp11 using Geometric Unfolding and Hierarchically Restrained Replica Exchange Molecular Dynamics,"</a> A Kumar, P Campitelli, SB Ozkan, MF Thorpe, Biophysical Journal 108 (2), 519a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(14)03298-6.pdf" target="_blank">"A Hinge Migration Mechanism Unlocks the Evolution of Green-To-Red Photoconversion in GFP-Like Proteins,"</a> RM Wachter, H Kim, T Zou, SB Ozkan, Biophysical Journal 108 (2), 380a-381a</p>
                    <p><a href="https://www.cell.com/structure/pdf/S0969-2126(14)00396-7.pdf" target="_blank">"A hinge migration mechanism unlocks the evolution of green-to-red photoconversion in GFP-like proteins,"</a> H Kim, T Zou, C Modi, K Dörner, TJ Grunkemeyer, L Chen, R Fromme, SB Ozkan, Structure 23 (1), 34-43</p>
                    <p>"Partial unfolding and refolding for structure refinement," A Kumar, P Campitelli, MF Thorpe, SB Ozkan, Proteins: Structure, Function and Bioinformatics</p>
                    <p><a href="https://academic.oup.com/mbe/article/32/1/132/2925568" target="_blank">"Evolution of conformational dynamics determines the conversion of a promiscuous generalist into a specialist enzyme,"</a> T Zou, VA Risso, JA Gavira, JM Sanchez-Ruiz, SB Ozkan, Molecular Biology and Evolution 32 (1), 132-143</p>
                </div>
                <div class="year-header">2014</div>
                <div class="publication">
                    <p><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0112701" target="_blank">"Proteome folding kinetics is limited by protein halflife,"</a> T Zou, N Williams, SB Ozkan, K Ghosh, PLoS One 9 (11), e112701</p>
                    <p>"Antiviral lectin cyanovirin-N: Probing multivalency and glycan recognition through experimental and computational approaches," G Ghirlanda, B Woodrum, J Maxwell, A Bolia, SB Ozkan, Abstracts of Papers of the American Chemical Society 248</p>
                    <p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/jnc.12741" target="_blank">"The protein interacting with C‐kinase (PICK 1) interacts with and attenuates parkin‐associated endothelial‐like (PAEL) receptor‐mediated cell death,"</a> P Dutta, KE O'Connell, SB Ozkan, AW Sailer, KK Dev, Journal of Neurochemistry 130 (3), 360-373</p>
                    <p><a href="https://pubs.acs.org/doi/full/10.1021/bi401651e" target="_blank">"Adenine Nucleotide Translocase Is Acetylated in Vivo in Human Muscle: Modeling Predicts a Decreased ADP Affinity and Altered Control of Oxidative …,"</a> C Mielke, N Lefort, CG McLean, JM Cordova, PR Langlais, AJ Bordner, SB Ozkan, Biochemistry 53 (23), 3817-3829</p>
                    <p><a href="https://pubs.acs.org/doi/abs/10.1021/ci4004927" target="_blank">"BP-Dock: a flexible docking scheme for exploring protein–ligand interactions based on unbound structures,"</a> A Bolia, ZN Gerek, SB Ozkan, Journal of Chemical Information and Modeling 54 (3), 913-925</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(14)00142-8.pdf" target="_blank">"A flexible docking scheme efficiently captures the energetics of glycan-cyanovirin binding,"</a> A Bolia, BW Woodrum, A Cereda, MA Ruben, X Wang, SB Ozkan, Biophysical Journal 106 (5), 1142-1151</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(13)04928-X.pdf" target="_blank">"Mechanistic Insights of β-Lactmases Evolution,"</a> T Zou, VA Risso, JA Gavira, JM Sanchez-Ruiz, SB Ozkan, Biophysical Journal 106 (2), 663a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(13)03873-3.pdf" target="_blank">"Biophysical Insights of Neutral and Non-Neutral Sequence Variants in the Human Proteome,"</a> BM Butler, S Kumar, SB Ozkan, Biophysical Journal 106 (2), 462a</p>
                </div>
                <div class="year-header">2013</div>
                <div class="publication">
                    <p><a href="https://meetings.aps.org/Meeting/4CF13/Session/C3.2" target="_blank">Designing Foldable Protein Sequence Through Zipping Contacts</a>, S Ozkan, Bulletin of the American Physical Society 58</p>
                    <p><a href="https://portlandpress.com/biochemsoctrans/article-abstract/41/5/1170/66328/The-antiviral-lectin-cyanovirin-N-probing" target="_blank">The antiviral lectin cyanovirin-N: probing multivalency and glycan recognition through experimental and computational approaches</a>, BW Woodrum, JD Maxwell, A Bolia, SB Ozkan, G Ghirlanda, Biochemical Society Transactions 41 (5), 1170-1176</p>
                    <p><a href="https://pubs.acs.org/doi/abs/10.1021/cb400030n" target="_blank">Electrostatic effect of the ribosomal surface on nascent polypeptide dynamics</a>, AM Knight, PH Culviner, N Kurt-Yilmaz, T Zou, SB Ozkan, S Cavagnero, ACS chemical biology 8 (6), 1195-1204</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(12)02537-4.pdf" target="_blank">Novel Physics-Based Protein Structure Refinement Method</a>, A Kumar, MF Thorpe, SB Ozkan, Biophysical Journal 104 (2), 229a</p>
                    <p><a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=Y4mtEasAAAAJ&cstart=20&pagesize=80&sortby=pubdate&citation_for_view=Y4mtEasAAAAJ:HDshCWvjkbEC" target="_blank">Structural dynamics flexibility informs function and evolution at a proteome scale</a>, ZN Gerek, S Kumar, SB Ozkan, Biophysical Journal 104 (2), 228a</p>
                </div>
                <div class="year-header">2012</div>
                <div class="publication">
                    <p>Dissecting the Determinants for Dimmanose Binding in Cyanovirin-N, B Woodrum, A Bolia, A Cereda, X Wang, SB Ozkan, G Ghirlanda, PROTEIN SCIENCE 21, 132-133</p>
                    <p><a href="https://iopscience.iop.org/article/10.1088/1478-3975/9/3/036004/meta" target="_blank">Dissipative electro-elastic network model of protein electrostatics</a>, DR Martin, SB Ozkan, DV Matyushov, Physical biology 9 (3), 036004</p>
                    <p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.24034" target="_blank">The binding affinities of proteins interacting with the PDZ domain of PICK1</a>, A Bolia, ZN Gerek, O Keskin, S Banu Ozkan, KK Dev, Proteins: Structure, Function, and Bioinformatics 80 (5), 1393-1408</p>
                    <p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002428" target="_blank">Collective dynamics differentiates functional divergence in protein evolution</a>, TJ Glembo, DW Farrell, ZN Gerek, MF Thorpe, SB Ozkan, PLoS computational biology 8 (3), e1002428</p>
                    <p>Protein evolution and foldability by conformational dynamics, SB Ozkan, ABSTRACTS OF PAPERS OF THE AMERICAN CHEMICAL SOCIETY 243</p>
                </div>
                <div class="year-header">2011</div>
                <div class="publication">
                    <p><a href="https://iopscience.iop.org/article/10.1088/1478-3975/8/6/066011/meta" target="_blank">Local and non-local native topologies reveal the underlying folding landscape of proteins</a>, T Zou, SB Ozkan, Physical biology 8 (6), 066011</p>
                    <p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002154" target="_blank">Change in allosteric network affects binding affinities of PDZ domains: analysis through perturbation response scanning</a>, ZN Gerek, SB Ozkan, PLoS computational biology 7 (10), e1002154</p>
                    <p>Perturbation Response Scanning Method for Identifying Allosteric Transitions and Utilizing in Flexible Docking, ZN Gerek, A Bolia, SB Ozkan, Biophysical Journal 100 (3), 372a-373a</p>
                    <p>Manipulation of conformational change in proteins by single-residue perturbations, C Atilgan, ZN Gerek, SB Ozkan, AR Atilgan, Biophysical journal 99 (3), 933-943</p>
                </div>
                <div class="year-header">2010</div>
                <div class="publication">
                    <p><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.366" target="_blank">A flexible docking scheme to explore the binding selectivity of PDZ domains</a>, ZN Gerek, SB Ozkan, Protein Science 19 (5), 914-928</p>
                    <p><a href="https://www.cell.com/fulltext/S0006-3495(09)01798-6" target="_blank">Union of geometric constraint-based simulations with molecular dynamics for protein structure prediction</a>, TJ Glembo, SB Ozkan, Biophysical journal 98 (6), 1046-1054</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(09)04304-5.pdf" target="_blank">Recreating the Structure and Dynamics of Ancestral Proteins</a>, T Glembo, M Thorpe, SB Ozkan, Biophysical Journal 98 (3), 460a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(09)01961-4.pdf" target="_blank">Application of Linear Response Theory on Protein Networks For Identifying Allosteric Transitions</a>, ZN Gerek, SB Ozkan, Biophysical Journal 98 (3), 26a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(09)05297-7.pdf" target="_blank">A Physics-Based Approach for Understanding Foldability</a>, T Zou, SB Ozkan, Biophysical Journal 98 (3), 637a-638a</p>
                </div>
                <div class="year-header">2009</div>
                <div class="publication">
                    <p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.22492" target="_blank">Identification of specificity and promiscuity of PDZ domain interactions through their dynamic behavior</a>, ZN Gerek, O Keskin, SB Ozkan, Proteins: Structure, Function, and Bioinformatics 77 (4), 796-811</p>
                    <p><a href="https://arxiv.org/abs/0907.3750" target="_blank">Essential sub-networks based on contact strength reveal folding kinetics</a>, T Zou, C Atilgan, AR Atilgan, SB Ozkan, arXiv preprint arXiv:0907.3750</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(08)00446-3.pdf" target="_blank">Blind test of physics-based prediction of protein structures</a>, MS Shell, SB Ozkan, V Voelz, GA Wu, KA Dill, Biophysical journal 96 (3), 917-924</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(08)03971-4.pdf" target="_blank">Pressure Induced Denaturation in Proteins: Stability and Kinetics</a>, K Ghosh, SB Ozkan, Biophysical Journal 96 (3), 569a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(08)03691-6.pdf" target="_blank">Protein Structure Refinement Using Physics-Based Models And Sampling</a>, SB Ozkan, X Ni, J Gee, MS Shell, Biophysical Journal 96 (3), 653a-654a</p>
                    <p><a href="https://www.cell.com/biophysj/pdf/S0006-3495(08)03365-1.pdf" target="_blank">Flexible Docking in PDZ domains using Elastic Network Model and Replica Exchange Molecular Dynamics</a>, ZN Gerek, SB Ozkan, Biophysical Journal 96 (3), 598a-599a</p>
                </div>
                <div class="year-header">2008</div>
                <div class="publication">
                    <p><a href="https://link.springer.com/article/10.1007/s00018-008-8122-2" target="_blank">Structure and dynamic regulation of Src-family kinases</a>, JR Engen, TE Wales, JM Hochrein, MA Meyn, S Banu Ozkan, I Bahar, Cellular and Molecular Life Sciences 65, 3058-3073</p>
                    <p><a href="https://www.annualreviews.org/content/journals/10.1146/annurev.biophys.37.092707.153558" target="_blank">The protein folding problem</a>, KA Dill, SB Ozkan, MS Shell, TR Weikl, Annu. Rev. Biophys. 37, 289-316</p>
                </div>
                <div class="year-header">2007</div>
                <div class="publication">
                    <p><a href="https://aiche.confex.com/aiche/2007/techprogram/P96860.HTM" target="_blank">Robust Design Of Carbon Catabolite Repression</a>, ZN Gerek, B Sariyar-Akbulut, SB Ozkan, The 2007 Annual Meeting</p>
                    <p><a href="https://pubs.acs.org/doi/abs/10.1021/ja066785b" target="_blank">The ultimate speed limit to protein folding is conformational searching</a>, K Ghosh, SB Ozkan, KA Dill, Journal of the American Chemical Society 129 (39), 11920-11927</p>
                    <p><a href="https://www.pnas.org/doi/abs/10.1073/pnas.0703700104" target="_blank">Protein folding by zipping and assembly</a>, SB Ozkan, GA Wu, JD Chodera, KA Dill, Proceedings of the National Academy of Sciences 104 (29), 11987-11992</p>
                    <p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0959440X07000772" target="_blank">The protein folding problem: when will it be solved?</a>, KA Dill, SB Ozkan, TR Weikl, JD Chodera, VA Voelz, Current opinion in structural biology 17 (3), 342-346</p>
                </div>
                <div class="year-header">2006</div>
                <div class="publication">
                    <p>Protein structure prediction by zipping and assembly, SB Ozkan, MS Shell, AG Wu, VA Voelz, KA Dill, BIOPHYSICAL JOURNAL, 535A-535A</p>
                </div>
                <div class="year-header">2005</div>
                <div class="publication">
                    <p><a href="#" target="_blank">Mining energy landscape parameters from two-state protein folding experiments</a>, SB Ozkan, K Ghosh, KA Dill, BIOPHYSICAL JOURNAL 88 (1), 40A-40A</p>
                </div>
                <div class="year-header">2004</div>
                <div class="publication">
                    <p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.10399" target="_blank">Conformational search of peptides and proteins: Monte Carlo minimization with an adaptive bias method applied to the heptapeptide deltorphin</a>, SB Ozkan, H Meirovitch, Journal of computational chemistry 25 (4), 565-572</p>
                </div>
                <div class="year-header">2003</div>
                <div class="publication">
                    <p><a href="https://pubs.acs.org/doi/abs/10.1021/jp0346615" target="_blank">Efficient conformational search method for peptides and proteins: Monte Carlo minimization with an adaptive bias</a>, SB Ozkan, H Meirovitch, The Journal of Physical Chemistry B 107 (34), 9128-9131</p>
                    <p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/bip.10280" target="_blank">Computing the transition state populations in simple protein models</a>, SB Ozkan, KA Dill, I Bahar, Biopolymers: Original Research on Biomolecules 68 (1), 35-46</p>
                </div>
                <div class="year-header">2002</div>
                <div class="publication">
                    <p><a href="https://onlinelibrary.wiley.com/doi/full/10.1110/ps.0207102" target="_blank">Fast‐folding protein kinetics, hidden intermediates, and the sequential stabilization model</a>, SB Ozkan, KA Dill, I Bahar, Protein Science 11 (8), 1958-1970</p>
                    <p>Unfolding events of chymotrypsin inhibitor 2 (CI2) revealed by Monte Carlo simulations, SG Kirca, SB Ozkan, T Haliloglu, BIOPHYSICAL JOURNAL 82 (1), 134A-134A, 2002</p>
                </div>
                <div class="year-header">2001</div>
                <div class="publication">
                    <p><a href="https://www.nature.com/articles/nsb0901-765" target="_blank">Transition states and the meaning of Φ-values in protein folding kinetics</a>, SB Ozkan, I Bahar, KA Dill, nature structural biology 8 (9), 765-769</p>
                </div>
                <div class="year-header">1999</div>
                <div class="publication">
                    <p>Identification of native conformations of proteins using a low resolution model, ŞB Özkan, Fen Bilimleri Enstitüsü</p>
                </div>
                <div class="year-header">1998</div>
                <div class="publication">
                    <p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/(SICI)1097-0134(19980801)32:2%3C211::AID-PROT7%3E3.0.CO;2-N" target="_blank">Recognition of native structure from complete enumeration of low-resolution models with constraints.</a>, B Ozkan, I Bahar, Proteins 32 (2), 211-222</p>
                </div>
    
            </div>
        </section>
        </main>
        <!--Footer-->
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
        <script src="../js/scripts.js"></script>
    </body>
</html>